cmd/adapter/config_test.go

```diff
@@ -0,0 +1,461 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/logrange/logrange/pkg/forwarder"
+	"github.com/logrange/logrange/pkg/utils"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/gravitational/logging-app/cmd/adapter/k8s"
+	"github.com/logrange/range/pkg/transport"
+)
+
+func TestLoadCfgFromFile(t *testing.T) {
+	t.Run("test1", func(t *testing.T) {
```

Comment:
nit: `test1` is a poor name for a test and does not reveal what's being tested.
Besides I think it's redundant as there's just one test.

Reply:
applied